samples/reactJS/src/components/app.jsx

```diff
@@ -3,9 +3,16 @@ import './app.css';
 import '../../../../src/webComponents/avatar';
 import '../../../../src/webComponents/smart-avatar';
 import React, {Component} from 'react';
+
 import SmartWebexTeamsAvatar from '../../../../src/reactComponents/smart-avatar';
 import { ToggleSwitch } from '@momentum-ui/react';
 
+/* eslint-disable sort-imports */
```

Comment:
this is going to bug me dude. can you just remove `sort-import` rule from `eslint` for me pretty please?